addons/spreadsheet/static/src/helpers/model.js

```diff
@@ -76,9 +77,49 @@ export async function freezeOdooData(model) {
             }
         }
     }
+    if (model.getters.getGlobalFilters().length === 0) {
+        return data;
+    }
+    data.sheets.push(exportGlobalFiltersToSheet(model, data));
     return data;
 }
 
+function exportGlobalFiltersToSheet(model, data) {
+    const styles = Object.entries(data.styles);
+    let titleStyleId =
+        styles.findIndex((el) => JSON.stringify(el[1]) === JSON.stringify({ bold: true })) + 1;
+
+    if (titleStyleId <= 0) {
+        titleStyleId = styles.length + 1;
+        data.styles[styles.length + 1] = { bold: true };
+    }
```

Comment:
I don't think you need to check if the style `{ bold: true }` already exists. Just add the style. It doesn't matter if it's duplicated. Keep it simple :)
```suggestion
    const titleStyleId = styles.length + 1;
    data.styles[styles.length + 1] = { bold: true };
```